### PR TITLE
fix spelling errors

### PR DESCRIPTION
**Pull request by zhaify**
recreate PR [17928](https://github.com/PaddlePaddle/Paddle/pull/17928)

### CODE REVIEW

File: paddle/fluid/API.spec [hunk 1]
```diff
@@ -523,7 +523,7 @@ paddle.fluid.LoDTensor.__init__ 1. __init__(self: paddle.fluid.core.LoDTensor, a
 paddle.fluid.LoDTensor.has_valid_recursive_sequence_lengths has_valid_recursive_sequence_lengths(self: paddle.fluid.core.LoDTensor) -> bool
 paddle.fluid.LoDTensor.lod lod(self: paddle.fluid.core.LoDTensor) -> List[List[int]]
 paddle.fluid.LoDTensor.recursive_sequence_lengths recursive_sequence_lengths(self: paddle.fluid.core.LoDTensor) -> List[List[int]]
```
Comment (luotao1) [inline paddle/fluid/API.spec]: please resolve conflict. and please do not change the API which you don't change in this PR.

**Comment by luotao1** [comment]
please resolve the conflict.

**Review by luotao1 — DISMISSED** [review]
LGTM

File: paddle/fluid/API.spec [hunk 2]
```diff
@@ -341,17 +341,17 @@ paddle.fluid.layers.uniform_random (ArgSpec(args=['shape', 'dtype', 'min', 'max'
...
+paddle.fluid.layers.ssd_loss (ArgSpec(args=['location', 'confidence', 'gt_box', 'gt_label', 'prior_box', 'prior_box_var', 'background_label', 'overlap_threshold', 'neg_pos_ratio', 'neg_overlap', 'loc_loss_weight', 'conf_loss_weight', 'match_type', 'mining_type', 'normalize', 'sample_size'], varargs=None, keywords=None, defaults=(None, 0, 0.5, 3.0, 0.5, 1.0, 1.0, 'per_prediction', 'max_negative', True, None)), ('document', '055bd5070ad72dccc0949b4ed036f39c'))
+paddle.fluid.layers.rpn_target_assign (ArgSpec(args=['bbox_pred', 'cls_logits', 'anchor_box', 'anchor_var', 'gt_boxes', 'is_crowd', 'im_info', 'rpn_batch_size_per_im', 'rpn_straddle_thresh', 'rpn_fg_fraction', 'rpn_positive_overlap', 'rpn_negative_overlap', 'use_random'], varargs=None, keywords=None, defaults=(256, 0.0, 0.5, 0.7, 0.3, True)), ('document', '70d0109c864bced99b6b0aca4574af5e'))
+paddle.fluid.layers.anchor_generator (ArgSpec(args=['input', 'anchor_sizes', 'aspect_ratios', 'variance', 'stride', 'offset', 'name'], varargs=None, keywords=None, defaults=(None, None, [0.1, 0.1, 0.2, 0.2], None, 0.5, None)), ('document', 'acc23232f4c8c03791598500b5bf7790'))
```
Comment (luotao1) [inline paddle/fluid/API.spec]:
```
[10:41:19]	[Step 1/1] API Difference is: 
[10:41:19]	[Step 1/1] - paddle.fluid.layers.anchor_generator (ArgSpec(args=['input', 'anchor_sizes', 'aspect_ratios', 'variance', 'stride', 'offset', 'name'], varargs=None, keywords=None, defaults=(None, None, [0.1, 0.1, 0.2, 0.2], None, 0.5, None)), ('document', 'acc23232f4c8c03791598500b5bf7790'))
[10:41:19]	[Step 1/1] + paddle.fluid.layers.anchor_generator (ArgSpec(args=['input', 'anchor_sizes', 'aspect_ratios', 'variance', 'stride', 'offset', 'name'], varargs=None, keywords=None, defaults=(None, None, [0.1, 0.1, 0.2, 0.2], None, 0.5, None)), ('document', 'acc23232f4c8c03791598500b5bf7790'))
```
http://ci.paddlepaddle.org/viewLog.html?buildId=116829&buildTypeId=YYTest_PrCiCpuPy2&tab=buildLog
api的位置顺序不要挪到，辛苦再修改一下。

Comment (zhaify) [reply paddle/fluid/API.spec]: 已经修改，不知道API 顺序的改变也会影响CI，我之前对比了md5，没找到问题，多谢提醒。

**Review by luotao1 — APPROVED** [review]

**Review by xsrobin — APPROVED** [review]

**Comment by luotao1** [comment]
@zhaify Could you cherry-pick into release/1.5? Thanks very much!
Note that api.spec between develop and release/1.5 maybe different.

**Pull request merged**